prow/gerrit/adapter/adapter.go

```diff
@@ -458,8 +477,12 @@ func (c *Controller) processChange(logger logrus.FieldLogger, instance string, c
 		return "", nil
 	}
 
+	logger.WithField("duration(s)", time.Since(tracker).Seconds()).Debug("Ready to process gerrit changes.")
```

Comment:
nit: Its unclear to me what this is measuring from the log message.

Reply:
updated